### DISCUSSION TITLE
Help in issues with _grid.scss row > * vs Col specificity when applied to angular 14 application

### DISCUSSION THREAD

**Discussion by jhasebo** [twbs/bootstrap#38937] [Q&A]
Would like to ask for assistance on this issue. Upgrading from version 4.1.3 to 5.3.0 breaks our bootstrap grid. Row > * width 100% is being applied over column widths.

![image](https://github.com/twbs/bootstrap/assets/10172310/13cc6dcb-3ac9-4fc5-b925-d0b66be3bb71)

**Comment by sidh41** [comment]
Have you found any solution to this?
